python/src/opendp/extras/numpy/_make_np_sum/__init__.py

```diff
@@ -20,6 +20,12 @@ def make_np_sum(input_domain: Domain, input_metric: Metric) -> Transformation:
 
     dp.assert_features("contrib", "floating-point")
 
+    if not str(input_domain).startswith("NPArray2Domain"):
+        raise ValueError("input_domain must be NPArray2Domain")  # pragma: no cover
+    
+    if input_domain.descriptor.nan:
+        raise ValueError("input_domain must not permit NaN elements")  # pragma: no cover
```

Comment:
Ditto: Prefer to always include what we got instead, and to try to have test coverage, unless this is a branch we shouldn't go down unless there's a bug.